zmk/templates/board/nrf52840/common_outer.dtsi

```diff
@@ -14,46 +14,50 @@
     status = "okay";
 };
 
-&usbd {
-    status = "okay";
+&uart0 {
+    compatible = "nordic,nrf-uarte";
+    current-speed = <115200>;
+    pinctrl-0 = <&uart0_default>;
+    pinctrl-1 = <&uart0_sleep>;
+    pinctrl-names = "default", "sleep";
+};
```

Comment:
This is present in all the current nRF52 boards in ZMK, but I'm actually not sure if this is needed for anything.

Reply:
I'm not sure it is currently, but I think it's useful to leave in to make wired split in the future easier.